roles/sap_hana_rhv_hypervisor/tasks/assert-configuration.yml

```diff
@@ -0,0 +1,127 @@
+---
+# tasks file for sap_hana_rhv_hypervisor
+
+- name: Get kernel command line
+  command: cat /proc/cmdline
+  register: __sap_hana_rhv_hypervisor_kernelcmdline_assert
+
+- name: "Assert - Kernel same page merging (KSM): Get status"
+  shell: systemctl status ksm
+  register: __sap_hana_rhv_hypervisor_ksmstatus_assert
+  ignore_errors: yes
+
+- name: "Assert - Kernel same page merging (KSM): Check if stopped"
+  assert:
+    that: "'Active: inactive (dead)' in __sap_hana_rhv_hypervisor_ksmstatus_assert.stdout"
+    fail_msg: "FAIL: ksm is running"
+    success_msg: "PASS: ksm not running"
+  ignore_errors: "{{ sap_hana_rhv_hypervisor_ignore_failed_assertion }}"
+
+- name: "Assert - Kernel same page merging (KSM) Tuning Daemon: Get status"
+  shell: systemctl status ksmtuned
```

Comment:
can be substituted for command, does not actually use any shell functions like piping